src/main/java/com/google/cloud/tools/appengine/api/AppEngineException.java

```diff
@@ -16,8 +16,8 @@
 
 package com.google.cloud.tools.appengine.api;
 
-/** API-level runtime exception for wrapping any failures in performing library operations. */
-public class AppEngineException extends RuntimeException {
+/** API-level exception for wrapping any failures in performing library operations. */
+public class AppEngineException extends Exception {
```

Comment:
do we need this exception at all?  When would you want to catch this instead of a more specific error that you can handle?

Reply:
Right now there are about 20 places where we throw a raw AppEngineException instead of a subclass. Might be worth looking at making this abstract in another PR if we shouldn't be doing that.